package.json

```diff
@@ -9,38 +9,32 @@
         "test": "npm run test:node && npm run test:browser",
         "test:browser": "node node_modules/testem/testem.js ci --file tests/testem.js",
         "test:node": "node node_modules/nyc/bin/nyc.js node tests/js/all-tests.js --report none --print none",
-        "posttest:cleanup": "node tests/js/pouchdb-express-tests/final-cleanup.js",
-        "posttest:report": "node node_modules/nyc/bin/nyc.js report",
-        "posttest": "npm run posttest:cleanup && npm run posttest:report"
+        "posttest": "node node_modules/nyc/bin/nyc.js report"
     },
     "dependencies": {
         "body-parser": "1.18.3",
-        "express": "4.16.3",
-        "@the-t-in-rtf/express-pouchdb": "4.0.0-20180523-1217",
-        "gpii-express": "1.0.13",
+        "express": "4.16.4",
+        "gpii-express": "1.0.15",
         "graceful-fs": "4.1.11",
-        "infusion": "3.0.0-dev.20180222T160835Z.6e1311a",
-        "lie": "3.2.0",
-        "memdown": "2.0.0",
-        "pouchdb": "6.4.3",
+        "infusion": "3.0.0-dev.20181024T171748Z.1fa91bed9",
+        "lie": "3.3.0",
```

Comment:
I can't see where this dependency is used

Reply:
Yup, it's gone.